File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/domain/interactor/error/ConfirmCodeValidationError.java
package com.lattis.ellipse.domain.interactor.error;

import java.util.ArrayList;
import java.util.List;

public class ConfirmCodeValidationError extends Throwable {

    private List<Status> status = new ArrayList<>();

    public ConfirmCodeValidationError(List<Status> status) {
        this.status = status;
    }

    public List<Status> getStatus() {
        return status;
    }

    public enum Status {
        INVALID_CONFIRMATION_CODE
    }

}

File: velolabs_repos/lattis_android_v2/domain/src/main/java/com/lattis/domain/error/ConfirmCodeValidationError.kt
package com.lattis.domain.error

import java.util.*

class ConfirmCodeValidationError(status: List<Status>) :
    Throwable() {
    var status: List<Status> = ArrayList()

    enum class Status {
        INVALID_CONFIRMATION_CODE
    }

    init {
        this.status = status
    }
}